Libraries/oneTBB/tbb-resumable-tasks-sycl/src/tbb-resumable-tasks-sycl.cpp

```diff
@@ -0,0 +1,137 @@
+//==============================================================
+// Copyright © 2019 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+
+#include <cmath>
+#include <array>
+#include <thread>
+#include <iostream>
+#include <atomic>
+#include <sstream>
+
+#include <CL/sycl.hpp>
+
+#include <tbb/blocked_range.h>
+#include <tbb/global_control.h>
+#include <tbb/task.h>
+#include <tbb/task_group.h>
+#include <tbb/parallel_for.h>
+
+constexpr cl::sycl::access::mode sycl_read = cl::sycl::access::mode::read;

```

Comment:
With the release of Beta09, the accessors should be updated to the more concise DPC++/SYCL 2020 form.

Reply:
If so I guess you need to use `#include <SYCL/sycl.hpp>` instead?